examples/.scene-tests

```diff
@@ -38,8 +38,4 @@ ignore "Tutorials/Mappings/TutorialMappingOctopusArticulated.scn"
 # uses the SofaCUDA plugin.
 ignore "Benchmark/Accuracy/cylinder_TLEDTetraSolution.scn"
 
-# is  only working with FlowVR
-ignore "Components/forcefield/DistanceGridForceField-liver.scn"
```

Comment:
I have the same question... it was supposed to be ignored so it shouldn't be executed in the CI ever.

Reply:
It was ignored because FlowVR was missing. The scene is now in a plugin and if we build this plugin in option mode only we can enable it (FlowVR is built in option mode).

Reply:
Anyway, the line in `.scene-tests` is useless since the scene is not there anymore.